lib/rules/lighter-http.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview Rule to disallow importing axios
+ * @author Julia Ziębińska
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const moduleVisitor = require("eslint-module-utils/moduleVisitor");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('../shared/types').Rule} */
+module.exports = {
+    meta: {
+        type: "problem",
+
+        docs: {
+            description: "disallow importing axios",
+            category: "Possible Errors",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/lighter-http"
+        },
+        messages: {
+            unexpected: "Do not import {{name}}. Instead use fetch."
```

Comment:
"Do not import {{name}}. Use fetch instead."

Reply:
"Do not import {{name}}. Use fetch instead (you can find examples on www.ec0lint.com).